website/dynamic-blocks/code-example.js

```diff
@@ -107,31 +208,76 @@ export const CodeExample = {
 			</Fragment>
 		);
 	},
-	component: ({ codeExample, showCode, showDemo = false, context }) => {
+	component: ({
+		codeExample,
+		showCode,
+		showDemo = false,
+		context,
+		heading = '',
+		size = 6,
+		level = 'h2',
+		addTableContent = false,
+		subText = '',
+	}) => {
 		if (typeof window === 'undefined') {
 			return <p>Loading...</p>;
 		}
+
+		const mq = useMediaQuery();
+		const id = heading.replace(/ /g, '-').toLowerCase();
 		const loadCodeBlock = codeExamples[codeExample];
 
 		return (
-			<Container css={blocksContainerStyle}>
-				<Grid columns={12}>
-					<Cell width={[12, 12, 12, 10, 10]} left={[1, 1, 1, 2, 2]}>
-						<Suspense fallback={<p>Loading...</p>}>
-							{loadCodeBlock && typeof window !== 'undefined' ? (
-								<ShowCodeBlock
-									loadCodeBlock={loadCodeBlock}
-									context={context}
-									showCode={showCode}
-									showDemo={showDemo}
-								/>
-							) : (
-								'Example not found.'
-							)}
-						</Suspense>
-					</Cell>
-				</Grid>
-			</Container>
+			<Cell
+				width={[12, 12, 12, 10, 10]}
+				left={[1, 1, 1, 2, 2]}
+				css={mq({
+					marginBottom: ['30px', null, null, null, '48px'],
+					height: 'auto',
+					':last-child': { marginBottom: 0 },
+				})}
+			>
+				<div css={mq({ marginBottom: ['18px', null, null, null, '24px'] })}>
+					{heading && (
+						<Heading
+							id={id}
+							tabIndex="-1"
+							tag={level}
+							size={[7, 7, 7, 7, size]}
```

Comment:
`size={[7, null, null, null, size]}` ?

Reply:
Can't do this yet until your heading changes are merged. Will go back and fix later.